aos_aoclc/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Dockerfile to execute a base machine with all set to test active optics
 # system software with the needed AOS repositories
 
-FROM lsstts/aos_sal:w_2020_14_sal
+FROM lsstts/aos_sal:w_2020_15_sal
```

Comment:
Any reason for not using a build argument for the image tag?

```
ARG image_tag

FROM lsstts/aos_aoclc:${image_tag}
```

Reply:
Done in the commit: 75bc978. Thanks!